source/reference/tables.txt

```diff
@@ -0,0 +1,132 @@
+.. _tables-reference:
+
+======
+Tables
+======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/styles/corrections.rst
+
+Tables compare items across multiple domains.
+Use tables whenever you need to compare multiple items in multiple ways.
+
+Table Types
+-----------
+
+rST offers multiple kinds of table, but ``list-table`` is the only
+variant we use at MongoDB.
+
+Introduce Tables
+----------------
+
+Tables represent lots of information simultaneously. To help your readers
```

Comment:
Suggestion: Somewhere on this page (possibly here), I think you should explicitly callout that you can have any number of rows and columns and how each group of bullets after need to match that (even if some are blank).

Reply:
Added during the discussion of row formatting.